documentation/src/main/asciidoc/chapter-10-advanced-mapping-options.asciidoc

```diff
@@ -344,6 +344,8 @@ public class CarMapperImpl implements CarMapper {
 
         if ( isNotEmpty( car.getOwner() ) ) {
             carDto.setOwner( car.getOwner() );
+        } else {
+            carDto.setOwner( null );
```

Comment:
Are you sure that we create this branch? I don't think that we do this for normal mappings?

Reply:
Indeed you are right. so that bit shouldn't be there, but perhaps there should be an example with an update method instead as well.